src/Alert/AlertButtons.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Button from '../Button';
+import { alertButtonProps } from './propTypes';
+
+const AlertButtons = ({
+  button,
+  className,
+  closeLabel,
+  dismissible,
+  onClose,
+}) => {
+  const buttonSize = 'sm';
+
+  const [ctaButtonText, ctaButtonAttrs] = useMemo(
+    () => {
+      const buttonAttrs = {};
+      const buttonText = button?.buttonText;
+      if (button) {
+        const {
+          className: ctaButtonClassName,
+          ...restButtonAttrs
+        } = button;
+        delete restButtonAttrs.buttonText;
+        Object.assign(buttonAttrs, {
+          size: buttonSize,
+          className: classNames(ctaButtonClassName, { 'ml-3': dismissible }),
+          ...restButtonAttrs,
+        });
+      }
+      return [buttonText, buttonAttrs];
+    },
+    [JSON.stringify(button)],
+  );
+
+  return (
+    <div className={classNames('pgn__alert-buttons d-flex justify-content-end', className)}>
```

Comment:
Let's use `ActionRow` here instead. Also, we should avoid using css utility classes in Paragon components since they have `!important` flags on them. See: https://github.com/edx/paragon/blob/master/docs/decisions/0012-css-styling-conventions#L21

Reply:
Ah, yes. Didn't think to use `ActionRow` here :) And +1 on avoiding the utility classes; will update.